src/Components/Details/Details.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import './Details.css'
+import backArrow from './double-left-arrows.svg'
+
+const Details = ({ allMovies, selectedMovieID, hideSelection }) => {
+
+  const movieToDisplay = allMovies.find(movie => {
+    return movie.id === selectedMovieID
+  })
+
+  console.log('MOVIE >>> ', movieToDisplay)
+
+  const releaseYear = movieToDisplay.release_date.split('-')[0]
+  const rating = movieToDisplay.average_rating.toFixed(1)
+  const genres = movieToDisplay.genres.map(genre => {
+  return <p className='details__each-genre'>{genre}</p>
+  })
+
+  const clearID = () => {
+    hideSelection(null)
+  }
+
+  return (
+    // <>
+    <section className='details__details-view'>
+      <button className='return-to-home-view-btn' onClick={clearID}>
+        <img src={backArrow} alt='back arrow icon' className='details__back-arrow'/>
+        {'All Movies'}
+      </button>
+      <article className='details__backdrop'>
+       <img 
+        className='details__backdrop-img'
+        src={movieToDisplay.backdrop_path}
+        alt={`${movieToDisplay.title} movie backdrop`}
+        />
+        <h1 className='details__movie-title'>{movieToDisplay.title}</h1>
+      </article>
+      <article className='details__movie-specs'>
+        <p className='details__movie-rating'>
+          {`Tater Rating: ${rating}`}
+        </p>
+        <p className='details__movie-release'>
+          {`${releaseYear} | ${movieToDisplay.runtime} mins`}
+        </p>
+        {/* <p className='details__movie-runtime'>
```

Comment:
Same  with removing comments before pushing - I've heard its a good practice to get those out before sending up changes.

Reply:
+1 to this :) I know it's scary to delete code! But it's better to just delete it and trust that you can write it again if ya need it